package.json

```diff
@@ -139,4 +143,4 @@
     "react": ">16",
     "framer-motion": ">1.6"
   }
-}
+}
```

Comment:
NBD: I'm assuming this was an inadvertent change; apparently I never set up an `.editorconfig` for Space Kit 🤷‍♂️